jinad/main.py

```diff
@@ -2,33 +2,57 @@
 import asyncio
 from typing import Any
 
+from fastapi import FastAPI
 from hypercorn.config import Config
 from hypercorn.asyncio import serve
-from fastapi import FastAPI
 
-from api.api import api_router
+from api.endpoints import flow, pod
+from config import fastapi_config, hypercorn_config, openapitags_config
 
-app = FastAPI(
-    title='Jina Flow Manager',
-    description='REST API for creating / deleting Jina Flow',
-    version='0.1.0'
-)
-# TODO: handle prefix
-app.include_router(router=api_router, prefix='/v1')
 
+def main(invocation: str = 'flow'):
```

Comment:
what does this mean? a `jinad` will be used either to spin Flows or Pods but not both?

Reply:
I do not see it clearly, but it seems that the `invocation = 'pod'` will be called from the API with `invocation='flow'`

Reply:
`jinad` would be running locally (for `flow invocation`) & as well at remote (for `pod invocation`).

The workflow is - 

- We have an IP `IP1` (_localhost or remote_). This machine has jinad running in a `flow` context.
- We have another IP `IP2` (_remote_) where jinad runs in a `pod` context.
- dashboard talks to an `IP1` to start a flow with different pods. If one of the pod has `host=IP2`, this flow triggers the pod api endpoint running on `IP2`

Reply:
ok, can't `jinad` live at the same time in both modes?

Reply:
If required, we can change this. For now, it is exclusive.